modules/manage/partials/tiered-storage.adoc

```diff
@@ -1268,6 +1312,13 @@ To create a new topic using remote recovery and enable Tiered Storage on the new
 rpk topic create <topic_name> -c redpanda.remote.recovery=true -c redpanda.remote.write=true -c redpanda.remote.read=true
 ----
 
+=== Performing rolling restart
```

Comment:
@andrwng Is the content on [this page](https://docs.redpanda.com/current/upgrade/rolling-upgrade/#perform-a-rolling-upgrade) the same as a "rolling restart"? How do we know when this step is required as part of recovery?

Reply:
Yeah, but without the "Upgrade your version" step